index.html

```diff
@@ -0,0 +1,231 @@
+             <a href="twitter.com">
+             <img src="./assets/pic2.svg" alt="">
+                
+             </a>
+        </div>
+            
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="cards__card cards__card--card1 ">
+      <div class="cards__card cards__card--img">
+
+        <img  class="cards__image" src="./assets/user-cover-2.svg" alt="">
+        <div class="cards__lower-box">
+          <h5 class="cards__names">
+                       Julian Jameson
+          </h5>
+
+          <figcaption class="cards__capt">Proffession</figcaption>
+
+          <div class="icons">
+
+          <div class="icons">
+          
+            <a href="facebook.com">
+                <img src="./assets/pic.svg" alt="">
+            </a> 
+  
+             <a href="instagram.com">
+              <img src="./assets/pic1.svg" alt="">
+                
+             </a>
+  
+             <a href="twitter.com">
+             <img src="./assets/pic2.svg" alt="">
+                
+             </a>
+        </div>
+            
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="cards__card cards__card--card1 ">
+      <div class="cards__card cards__card--img">
+
+        <img  class="cards__image" src="./assets/user-cover-3.svg" alt="">
+        <div class="cards__lower-box">
+          <h5 class="cards__names">
+                       Julian Jameson
+          </h5>
+
+          <figcaption class="cards__capt">Proffession</figcaption>
+
+          <div class="icons">
+
+          <div class="icons">
+          
+            <a href="facebook.com">
+                <img src="./assets/pic.svg" alt="">
+            </a> 
+  
+             <a href="instagram.com">
+              <img src="./assets/pic1.svg" alt="">
+                
+             </a>
+  
+             <a href="twitter.com">
+             <img src="./assets/pic2.svg" alt="">
+                
+             </a>
+        </div>
+            
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="cards__card cards__card--card1 ">
+      <div class="cards__card cards__card--img">
+
+        <img  class="cards__image" src="./assets/user-cover-4.svg" alt="">
+        <div class="cards__lower-box">
+          <h5 class="cards__names">
+                       Julian Jameson
+          </h5>
+
+          <figcaption class="cards__capt">Proffession</figcaption>
+
+          <div class="icons">
+
+          <div class="icons">
+          
+             <a href="facebook.com">
+              <img src="./assets/pic.svg" alt="">
+          </a> 
+
+           <a href="instagram.com">
+            <img src="./assets/pic1.svg" alt="">
+              
+           </a>
+
+           <a href="twitter.com">
+           <img src="./assets/pic2.svg" alt="">
+              
+           </a>
+        </div>
+            
+          </div>
+        </div>
+      </div>
+    </div>
+    </div>
+</div>
+
+
+
+
+<div class="page4">
```

Comment:
make use of HTML5 semantics to provide meaning to your HTML page(such as footer, section ...)